Cargo.toml

```diff
@@ -56,6 +56,7 @@ scl = "0.0.1"
 heck = "0.3.0"
 zip = "0.5.0"
 tar = "0.4.20"
+unidiff = { git = "https://github.com/foundpatterns/difflib" }
```

Comment:
It might be best to use a specific commit just in case there is a breaking change in future commits, it wont impact torchbear, but no need to worry about that for now.

Reply:
noted